packages/EasyErrorHandler/src/Bridge/Symfony/Providers/ErrorCodesProvider.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Symfony\Providers;
+
+use EonX\EasyErrorHandler\Bridge\Symfony\Interfaces\ErrorCodes\ErrorCodesProviderInterface;
+use ReflectionClass;
+use ReflectionException;
+use Symfony\Component\ErrorHandler\Error\ClassNotFoundError;
+
+final class ErrorCodesProvider implements ErrorCodesProviderInterface
+{
+    private const ERROR_CODE_NAME_PREFIX = 'ERROR_';
+
+    /**
+     * @param class-string $errorCodesInterface
+     */
+    public function __construct(private string $errorCodesInterface)
```

Comment:
Let's pass here a service that implements `ErrorCodesInterface`.
In the app, we can redefine this service.

I think it is better than binding variable `$errorCodesInterface`.

Reply:
We won't implement this interface in a service. The idea is to have `ErrorCodesInterface extends <EasyErrorHandler>ErrorCodesInterface` in the app. But I agree that binding this variable explicitly is not a good idea.

Reply:
But what can we do other than binding?

Reply:
@alexndlm that means I would need to define `error_codes_interface` even if I don't want this feature.

Reply:
@akl-95 hmmmm, yes.